frontend/src/stories/Introduction.stories.mdx

```diff
@@ -4,12 +4,24 @@ import { Meta } from '@storybook/addon-docs';
 
 # Design System
 
-At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation in Angular or Rails.
+At OpenProject, we use a design system to ensure that our design delivers a consistent experience to users. The system describes the styles, components and patterns that come together to define the overall user interface. To do this, we provide documentation explaining how each element should be used, the different states, variations and options it offers, along with example implementation.
 
 Our goal is to reach a point where every view in OpenProject is built with elements described in the design system. However, we are aware that reaching this goal will take time for a tool as complex and layered as OpenProject.
```

Comment:
@psatyal Again, I would not go so far. I see no reason to define every special case here. Only, if we think that there is a general pattern that we want to see reused, I'd add it to the DS.

```suggestion
Our goal is to reach a point where parts of views in OpenProject is built with elements described in the design system. However, we are aware that reaching this goal will take time for a tool as complex and layered as OpenProject.
```

Reply:
I can see why "every view" is too strong,  but "parts of views" is too weak imo.

How about

> Our goal is to reach a point where every view in OpenProject has been built with our design system principles and patterns in mind.

Reply:
Agree.